templates/partial/filter.html

```diff
@@ -18,18 +19,29 @@ <h2 class="govuk-heading-m">Selected filters</h2>
             <div class="moj-filter__heading-action">
               <p>
                 <a class="govuk-link govuk-link--no-visited-state"
-                href="{% url 'home:search' %}{% query_string domains=None %}"
+                href="{% url 'home:search' %}{% query_string domain=None classifications=None where_to_access=None %}"
                 id="clear_filter">
                 Clear filter
                 </a>
               </p>
             </div>
           </div>
-          {% if form.domains.value %}
+          {% if label_clear_href|get_item:"domain"|length > 0 %}
             <h3 class="govuk-heading-s govuk-!-margin-bottom-0">Domain</h3>
             <ul class="moj-filter-tags">
               <input type="hidden" name="clear_label" value="">
-              {% for label, href in label_clear_href.items %}
+              {% for label, href in label_clear_href|get_items:"domain" %}
```

Comment:
Suggestion: maybe we can move this into a partial and pass in the filter type "domain" or "classifications"

Reply:
Moved to partial and no longer repeats the same code for each field.